pkg/cli/flags.go

```diff
@@ -229,6 +229,46 @@ var clusterNameRe = regexp.MustCompile(`^[a-zA-Z](?:[-a-zA-Z0-9]*[a-zA-Z0-9]|)$`
 
 const maxClusterNameLength = 256
 
+type filter int8
```

Comment:
I wonder if we maybe should just use a `string` base type instead of `int8` for these. That would make `String()` and `Set()` trivial.